application/templates/static_site/measure.html

```diff
@@ -12,7 +12,7 @@
 
 {% block pageTitle %}{{ measure_version.title }} - GOV.UK Ethnicity facts and figures{% endblock %}
 {% block socialTitle %}{{ measure_version.title }}{% endblock %}
-{% block socialDescription %}{{ measure_version.summary | first_bullet | render_markdown | striptags }}{% endblock %}
+{% block socialDescription %}{% if measure_version.description %}{{ measure_version.description | striptags }}{% else %}{{ measure_version.summary | first_bullet | render_markdown | striptags }}{% endif %}{% endblock %}
```

Comment:
We shouldn't be saving tags in the description?  So probably shouldn't need to `striptags`?

Reply:
@TheDoubleK yeah, there _shouldn’t_ be any tags in there, but I figured it was probably best to be defensive and escape them anyway, to protect against a `<script>` tag or something being injected? Or alternatively someone using the `>` sign to mean "greater than" (which also seems fairly unlikely).

Reply:
One of the benefits of Jinja templating is that everything should be escaped by default. Are you sure this is actually a vulnerability? Normally unless you pass stuff through the `| safe` filter, it gets escaped automatically.

Reply:
Hmm, yeah makes sense and no harm in keeping it.

Reply:
Ah ok, if it's escaped automatically I think we can remove `striptags`. I'll do that.